tests/receivers/smartagent/collectd-activemq/testdata/resource_metrics/all.yaml

```diff
@@ -0,0 +1,81 @@
+resource_metrics:
```

Comment:
Is this a copy/paste from SA or net new? I don't know how to review this, but if it existed before and worked for us I can assume it is what we need.

Given how large this PR is and that parts of it are copy/pasted code can you comment on each file asking for a review if that file needs it. It will help reviewers.

Reply:
All the resource metrics file content is new, but contains all the metrics that were received when running the target service and collector.  The validity of their content needs to be determined once the include/exclude filter functionality is adopted.

Reply:
I've listed all files that are taken from the agent codebase